web/client/epics/contextcreator.js

```diff
@@ -90,10 +90,11 @@ export const saveContextResource = (action$, store) => action$
         const groups = groupsSelector(state);
         const backgrounds = backgroundListSelector(state);
         const textSearchConfig = textSearchConfigSelector(state);
+        const bookmarkSearchConfig = bookmarkSearchConfigSelector(state);
         const additionalOptions = mapOptionsToSaveSelector(state);
         const plugins = pluginsSelector(state);
 
-        const mapConfig = MapUtils.saveMapConfiguration(map, layers, groups, backgrounds, textSearchConfig, additionalOptions);
+        const mapConfig = MapUtils.saveMapConfiguration(map, layers, groups, backgrounds, textSearchConfig, bookmarkSearchConfig, additionalOptions);
```

Comment:
will this change need a dedicated unit test?

Reply:
I don't see any specific scenario requirements, hence not required. And it is covered as part of MapUtils